FirebaseAppDistribution/Sources/FIRAppDistributionAuthPersistence.m

```diff
@@ -111,14 +124,18 @@ + (BOOL)persistAuthState:(OIDAuthState *)authState error:(NSError **_Nullable)er
 
 + (NSMutableDictionary *)getKeyChainQuery {
   NSMutableDictionary *keychainQuery = [NSMutableDictionary
-      dictionaryWithObjectsAndKeys:(id)kSecClassGenericPassword, (id)kSecClass, @"OAuth",
-                                   (id)kSecAttrGeneric, @"OAuth", (id)kSecAttrAccount,
-                                   @"fire-fad-auth", (id)kSecAttrService,
-                                   (id)kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly,
-                                   (id)kSecAttrAccessible, nil];
+      dictionaryWithObjectsAndKeys:(id)kSecClassGenericPassword, (id)kSecClass,
```

Comment:
Sorry for another invasion. Do you think using a dictionary literal with a mutable copy would be a bit more readable? Something like:
```
NSMutableDictionary *keychainQuery =
[@{
    (id)kSecClass : (id)kSecClassGenericPassword, 
    (id)kSecAttrGeneric: kFIRAppDistributionAuthPersistenceErrorKeychainId
} mutableCopy];
```

Reply:
Great idea. Done.